pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -79,23 +77,12 @@ social:
   -# Map
   %section.map
     .wrapper
-      - unless @country == 'ma' # Hide map for Morocco because of sensitive border disputes misrepresented in map
```

Comment:
Moved this to a view - [ref](https://github.com/code-dot-org/code-dot-org/pull/54234/files#r1361280784)